Justfile

```diff
@@ -150,5 +149,10 @@ check-protocol-schema:
     env {{protocol_schema_env}} cargo test -p protocol-schema-check --profile dev-artifacts
     env {{protocol_schema_env}} cargo run -p protocol-schema-check --profile dev-artifacts
 
-check_build_public_libraries:
-    cargo check {{public_libraries}}
+publishable := "cargo metadata --format-version 1 | jq -r '.packages[] | select(.name | startswith(\"near-\")) | select(.publish == null or (.publish | length > 0)) | .name'"
```

Comment:
Rather than filtering by package name, use `cargo metadata --no-deps`?

Reply:
Oh yeah, that's probably more reliable